### PR TITLE
Normalizing validation

### PR DESCRIPTION
**Pull request by markcowl**

### CODE REVIEW

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by ogail** [comment]
LGTM

File: Tools/TestRules.ruleset [hunk 1]
```diff
@@ -4,14 +4,41 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```
Comment (yugangw-msft) [inline Tools/TestRules.ruleset]: Do you think it make senses to name the file as "RulesForGeneratedCode.ruleset"

Comment (markcowl) [inline Tools/TestRules.ruleset]: Not entirely - this is a combination of exclusions for generated code and for tests.

**Pull request merged**